profile/README.md

```diff
@@ -101,15 +101,6 @@ TODO add this back in when there is content
 
 </details>
 
-## Our open-sourced work
-
-[![test_track](https://github-readme-stats.vercel.app/api/pin/?username=Betterment&repo=test_track)](https://github.com/Betterment/test_track)
```

Comment:
Can we replace this with text links instead? The "popular repositories" list isn't always relevant or active, see `backbone.blazer` 😄

Reply:
Done!